app/models/pair_request.rb

```diff
@@ -2,17 +2,41 @@
 #
 # Table name: pair_requests
 #
-#  id          :bigint           not null, primary key
-#  duration    :float
-#  when        :datetime
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
-#  acceptor_id :integer
-#  author_id   :integer
+#  id         :bigint           not null, primary key
+#  duration   :float            not null
+#  status     :integer          default("pending"), not null
+#  when       :datetime         not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  author_id  :bigint           not null
+#  invitee_id :bigint           not null
+#
+# Indexes
+#
+#  index_pair_requests_on_author_id   (author_id)
+#  index_pair_requests_on_invitee_id  (invitee_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (author_id => users.id)
+#  fk_rails_...  (invitee_id => users.id)
 #
 class PairRequest < ApplicationRecord
   belongs_to :author, class_name: 'User'
-  belongs_to :acceptor, class_name: 'User', optional: true
+  belongs_to :invitee, class_name: 'User'
+
+  before_create { pending! }
```

Comment:
Also, I don't think this is needed (you can add a quick test for creating a new PairRequest and checking that the status is pending.

Reply:
Without the hook, the status will be set to "pending" if the status is unspecified on creation (ie. `nil`).

With your request above for a `before_create` hook, I figured you wanted it such that even if someone did specify the `status` ie:
```rb
PairRequest.create(
  status: "expired",
  # other attrs
)
```
It would still end up being created with the status == "pending". If that's not what you wanted, I can pull it out.

Reply:
oooooh. I think it should just use the db level status. There are going to be times when we want to create it with something other than pending. Good insight though!

Reply:
So, lets remove this `before_create`